instructure/tinymce-a11y-checker
# Make why popover 'x' receive focus

This updates the popover to use the latest api rather than
the older api for handling close buttons

closes CORE-1699

Test Plan:
  - Open up an a11y checker instance
  - Go to the question mark for why
  - Focus should go to the close button and be trapped
    within the popover
